scripts/faucet-deploy

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# shellcheck source=lib/common.sh
+. "${LIB_DIR:-$(cd "${0%/*}/../lib"&&pwd)}/common.sh"
+
+# Deploy Token Faucet (no solc optimization)
+dappBuild token-faucet
```

Comment:
Let's not build if we're not gonna deploy, i.e. move the build inside the conditional.

Reply:
if we don't build then I don't think we get the ABIs in the `out` folder which I feel is still important because those contracts are part of the deployment even if we aren't deploying then.